packages/components/src/Stepper/Stepper.stories.js

```diff
@@ -74,7 +74,6 @@ export const StepperSuccessfulWithoutTransition = () => (
 export const FormStepper = () => (
 	<Stepper.Form>
 		<Stepper.Form.Step.Validated title="I'm ok" />
-		<Stepper.Form.Step.Error title="I'm not ok" />
```

Comment:
why ?

Reply:
I don't want to document things that the design team does not have access to. 

On Figma there's no such thing as the Error step. Last I checked, they were not sure they'd want it.